core/tests/performance_framework/selenium_driver.py

```diff
@@ -0,0 +1,152 @@
+"""Selenium + browsermobproxy
+
+This file contains selenium + browsermobproxy to fetch performance data.
+"""
+
+import urlparse
+import os
+
+from browsermobproxy import Server
+from selenium import webdriver
+from xvfbwrapper import Xvfb
+
+# Help Needed:
+# Need to curl both these binaries and install them with the setup process.
```

Comment:
See manifest.json and scripts/install_third_party.py. We already have dependencies in third_party/ that are not JS stuff. You may need to modify install_third_party.sh, install_third_party.py and/or manifest.json appropriately. @MAKOSCAFEE may be able to give you some guidance.

Also, chromedriver _might_ already be present somewhere in ../node_modules as part of the protractor stuff. Up to you if you want to reuse that or just download it separately.